README.md

```diff
@@ -2,6 +2,19 @@
 
 a repo to practice clone, push, pull and handle merge conflicts.
 
+HEY ALL I NEED TO MAKE A PULL REQUEST TO DO SCREEN SHOTS FOR SOME WORK THING, ILL DELETE THIS WHEN IM DONE 
+
+BREAKING CHANGE
+
+COOL THING
+
+THING DONE USING OLD SYNTAX NOT OLD PATTERN
```

Comment:
Suggestion: you could destructure this a different way, here's a link to an article. [super cool ES7 thing](www.javascript.com)
```suggestion
THING REDONE USING COOL NEW THING
```